### ISSUE TITLE
New translation: {Burmese}

### ISSUE THREAD

**Issue by sithu015** [squidfunk/mkdocs-material#1669]
## Instructions

1. Check, if your language is already available: [here](https://bit.ly/3c2ox0C)
2. If it isn't, please translate the labels on the right:

``` jinja
{% macro t(key) %}{{ {
  "language": "my",
  "direction": "ltr",
  "clipboard.copy": "ကလစ်ဘုတ် သို့ ကူးယူရန်",
  "clipboard.copied": "ကလစ်ဘုတ် သို့ ကူယူပြီး",
  "edit.link.title": "ဤ စာမျက်နှာကို ပြင်ရန်",
  "footer.previous": "နောက်သို့",
  "footer.next": "ရှေ့သို့",
  "footer.title": "အောက်ခြေ",
  "header.title": "ခေါင်းပိုင်း",
  "meta.comments": "မှတ်ချက်များ",
  "meta.source": "ရင်းမြစ်",
  "nav.title": "လမ်းညွှန်",
  "search.config.lang": "my",
  "search.config.pipeline": "trimmer, stopWordFilter",
  "search.config.separator": "[\s\-]+",
  "search.placeholder": "ရှာရန်",
  "search.reset": "ရှင်းလင်း",
  "search.result.placeholder": "ရှာဖွေခြင်းစရန် စာရိုက်ပါ",
  "search.result.none": "တူညီသော စာရွက်စာတမ်းများ မရှိပါ",
  "search.result.one": "စာရွက်စာတမ်း ၁ ခု တူညီသည်",
  "search.result.other": "စာရွက်စာတမ်း # ခု တူညီသည်",
  "skip.link.title": "မာတိကာ သို့ သွားရန်",
  "source.link.title": "repository သို့ သွားရန်",
  "source.revision.date": "နောက်ဆုံး ထုတ်ပြန်ချက်",
  "tabs.title": "တက်များ",
  "toc.title": "ပါဝင်အကြောင်းအရာများ"
}[key] }}{% endmacro %}
```

Thanks!

**Comment by squidfunk** [comment]
Awesome, thanks!

**Comment by squidfunk** [comment]
Added in 481e511a. Unfortunately, search doesn't work, as Lunr.js doesn't support Burmese. There may be a configuration which could make it work (like using the Japanese stemmers for the Chinese languages), but I don't have the time to fiddle with this. If you find something that works, feel free to PR!

**Comment by squidfunk** [comment]
Released as part of 5.1.6
